packages/demo/src/components/Bandada/Bandada.tsx

```diff
@@ -1,24 +1,58 @@
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+
+import { ActionBox } from "../ActionBox/ActionBox";
+
 interface IBandadaProps {
   joinGroup: () => Promise<void>;
   generateGroupMerkleProof: () => Promise<void>;
 }
 
+const GENERATE_MERKLE_TREE_CODE = `import { initializeCryptKeeper } from "@cryptkeeperzk/providers";
+
+const client = initializeCryptKeeper();
+
+const connect = (async () => {
+  await client?.request({
+    method: RPCExternalAction.GENERATE_GROUP_MERKLE_PROOF,
+    payload: {
+      groupId: process.env.TEST_GROUP_ID!,
+    },
+  });`;
+
+const JOIN_GROUP_CODE = `import { initializeCryptKeeper } from "@cryptkeeperzk/providers";
+
+const client = initializeCryptKeeper();
+
+const connect = async () => {
+  await client?.request({
+    method: RPCExternalAction.JOIN_GROUP,
+    payload: {
+      groupId: process.env.TEST_GROUP_ID!,
+      apiKey: process.env.TEST_GROUP_API_KEY,
+      inviteCode: process.env.TEST_GROUP_INVITE_CODE,
+    },
+  })`;
+
 export const Bandada = ({ joinGroup, generateGroupMerkleProof }: IBandadaProps): JSX.Element => (
-  <div>
-    <h2>Generate bandada group proof</h2>
-
-    <div>
-      <button type="button" onClick={generateGroupMerkleProof}>
-        Generate Group Merkle Proof
-      </button>
-    </div>
-
-    <br />
-
-    <div>
-      <button type="button" onClick={joinGroup}>
-        Join test group
-      </button>
-    </div>
-  </div>
+  <Box
+    className="popup"
+    sx={{ p: 3, display: "flex", flexDirection: "column", alignItems: "center", justifyContent: "center" }}
+  >
+    <Typography variant="h6">Integration with Bandada service</Typography>
+
+    <ActionBox<undefined, void>
+      actionCode={GENERATE_MERKLE_TREE_CODE}
+      actionFn={generateGroupMerkleProof}
+      actionOption={undefined}
```

Comment:
You don't need to pass undefined as props

Reply:
Because there are some functions have no option value, but as well I'd need option to be still have a required param in the Action Box, that's why passing `undefined` would resolve this generic issue.